apps/angular/src/app/features/anime/anime-page/anime-page.component.ts

```diff
@@ -151,16 +150,16 @@ export class AnimePageComponent implements OnInit {
 	 * @param pagination Pagination: pageNumber and pageSize.
 	 * @param sorting Sorting: field and direction.
 	 * @param search Search.
-	 * @param type Anime type.
+	 * @param types Anime type.
 	 */
 	private createAnimeParams(pagination: PaginationParams, sorting: Sorting<AnimeSortingField>,
-		search?: string, type?: AnimeType[]): AnimeParams {
+		search?: string, types?: AnimeType[]): AnimeParams {
 		return {
 			pagination,
 			sorting,
 			filters: {
 				search: search ?? defaultParams.filters.search,
-				types: type ?? defaultParams.filters.types,
+				types: types ?? defaultParams.filters.types,
 			},
 		};
 	}
```

Comment:
I'm not requesting changes, just in case you don't know about this feature:
```suggestion
	private createAnimeParams(
                pagination: PaginationParams,
                sorting: Sorting<AnimeSortingField>,
		search: string = defaultParams.filters.search,
                types: AnimeType[] = defaultParams.filters.types,
         ): AnimeParams {
		return {
			pagination,
			sorting,
			filters: { search, types },
		};
	}
```
Typescript allows us to set default values for parameters if arguments are not passed. We can use this feature to avoid extra checks in code.